test/TesterInternal/General/ElasticPlacementTest.cs

```diff
@@ -64,7 +54,7 @@ public override TestingSiloHost CreateSiloHost()
             LogCounts(activationCounts);
             logger.Info("-----------------------------------------------------------------");
             AssertIsInRange(activationCounts[this.HostedCluster.Primary], perSilo, leavy);
-            AssertIsInRange(activationCounts[this.HostedCluster.Secondary], perSilo, leavy);
+            AssertIsInRange(activationCounts[this.HostedCluster.GetActiveSilos().Skip(1).First()], perSilo, leavy);
```

Comment:
similar to other comment, in these just use SecondarySilos instead of skipping 1 of all active